include/nighthawk/common/version.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "absl/strings/str_cat.h"
+
+namespace Nighthawk {
+
+const int MAJOR_VERSION{0};
+const int MINOR_VERSION{3};
+const int PATCH_VERSION{0};
+
+class VersionUtils {
```

Comment:
Sorry I wasn't clear enough. I was wondering if this even needs to be in a class? How about a plain top-level function?

Reply:
Ah, sure! How about 7de573f15e7d85ee12d8364d88880a7c18cf9e06?